tests/integration/targets/aci_access_spine_interface_profile/tasks/main.yml

```
@@ -0,0 +1,206 @@
+  - name: Change description of spine interface profile again (check_mode)
+    cisco.aci.aci_access_spine_interface_profile:
+      <<: *interface_policy_spine_profile_present
+      description: Ansible test spine interface profile
+    check_mode: yes
+    register: cm_add_spine_interface_profile_descr_again
+
+  - name: Change description of spine interface profile again (normal mode)
+    cisco.aci.aci_access_spine_interface_profile:
+      <<: *interface_policy_spine_profile_present
+      description: Ansible test spine interface profile
+    register: nm_add_spine_interface_profile_descr_again
+
+  - name: Verify add_spine_interface_profile_descr
+    assert:
+      that:
+      - cm_add_spine_interface_profile_descr is changed
+      - nm_add_spine_interface_profile_descr is changed
+      - cm_add_spine_interface_profile_descr_again is not changed
+      - nm_add_spine_interface_profile_descr_again is not changed
+
+  # ADD SPINE INTERFACE PROFILE AGAIN
+  - name: Add spine interface profile again with no description (check_mode)
+    cisco.aci.aci_access_spine_interface_profile: *interface_policy_spine_profile_present
+    check_mode: yes
+    register: cm_add_spine_interface_profile_again_no_descr
+
+  - name: Add spine interface profile again with no description (normal mode)
+    cisco.aci.aci_access_spine_interface_profile: *interface_policy_spine_profile_present
+    register: nm_add_spine_interface_profile_again_no_descr
+
+  - name: Verify add_spine_interface_profile_again_no_descr
+    assert:
+      that:
+      - cm_add_spine_interface_profile_again_no_descr is not changed
+      - nm_add_spine_interface_profile_again_no_descr is not changed
+
+  # QUERY ALL SPINE INTERFACE PROFILES
+  - name: Query all interface profiles (check_mode)
+    cisco.aci.aci_access_spine_interface_profile: &interface_policy_spine_profile_query
+      <<: *aci_info
+      state: query
+    check_mode: yes
+    register: cm_query_all_spine_interface_profiles
+
+  - name: Query all spine_interface_profiles (normal mode)
+    cisco.aci.aci_access_spine_interface_profile: *interface_policy_spine_profile_query
+    register: nm_query_all_spine_interface_profiles
+
+  - name: Verify query_all_spine_interface_profiles
+    assert:
+      that:
+      - cm_query_all_spine_interface_profiles is not changed
+      - nm_query_all_spine_interface_profiles is not changed
+
+  # QUERY A SPINE INTERFACE PROFILE
+  - name: Query our spine_interface_profile
+    cisco.aci.aci_access_spine_interface_profile:
+      <<: *interface_policy_spine_profile_query
+      spine_interface_profile: ansible_test
+    check_mode: yes
+    register: cm_query_spine_interface_profile
+
+  - name: Query our spine_interface_profile
+    cisco.aci.aci_access_spine_interface_profile:
+      <<: *interface_policy_spine_profile_query
+      spine_interface_profile: ansible_test
+    register: nm_query_spine_interface_profile
+
+  - name: Verify query_spine_interface_profile
+    assert:
```

```suggestion
     ansible.builtin.assert:
```
Plus all the other places that don't use the full module name.